src/restrictededitingmodeediting.js

```diff
@@ -282,9 +282,14 @@ export default class RestrictedEditingModeEditing extends Plugin {
 function getCommandExecuter( editor, commandName ) {
 	return ( data, cancel ) => {
 		const command = editor.commands.get( commandName );
+		const navigationCommands = commandName === 'goToPreviousRestrictedEditingException' ||
+			commandName === 'goToNextRestrictedEditingException';
 
 		if ( command.isEnabled ) {
 			editor.execute( commandName );
+		} else if ( navigationCommands ) {
```

Comment:
I don't get this change or maybe I'm missing something.

This helper method is used twice https://github.com/ckeditor/ckeditor5-restricted-editing/blob/24740bce1074fc2e35f7f592f54183b897cb4375/src/restrictededitingmodeediting.js#L84-L85  and exclusively for `goToPreviousRestrictedEditingException` and `goToNextRestrictedEditingException` only.

So filtering out anything here does not make sense because this helper:
* is not used anywhere else,
* it's unlikely to be used anywhere else in the future, and even if so, then will be the right moment to think about code branching inside of it. Check out [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

Correct me if I'm wrong but I think the whole fix should come down to cancelling the event (the final `cancel();`) only if the command was enabled. Something like

```js
// Helper method for executing enabled commands only.
function getCommandExecuter( editor, commandName ) {
	return ( data, cancel ) => {
		const command = editor.commands.get( commandName );

		if ( command.isEnabled ) {
			editor.execute( commandName );
			cancel();
		}
	};
}
```

Reply:
> Correct me if I'm wrong but I think the whole fix should come down to cancelling the event (the final cancel();) only if the command was enabled

Technically yes, but I still don't have full knowledge about how things work "around" that plugin, so I tried to do this as safely as I could ;-) 

But if you say, that I could safely move `cancel()` to condition' scope, then I'm good with that.